bin/smsd.py

```diff
@@ -171,16 +176,6 @@ def main():
 
         sys.exit(0)
 
-    # Switch user to $NAV_USER (only works if we're root)
```

Comment:
No longer needed?

Reply:
That's a better comment, yes. But the entire block of code that follows is gone.

Reply:
Sorry, I wasn't reading the same code you were, @hm. This entire code block was moved further _up_ - to ensure privileges are dropped before any log files are created.